packages/terra-clinical-result/src/proptypes/observationPropTypes.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import interpretationPropType from './interpretationPropTypes';
+import resultTypePropType from './resulttypePropTypes';
+import statusPropType from './statusPropTypes';
+import { valueQuantityPropShape, valueStringPropShape, valueNullPropShape } from './valuePropTypes';
+
+const observationPropShape = PropTypes.shape({
+  /**
+   *  Event ID for result
+   */
+  eventId: PropTypes.string,
+  /**
+   *  Value and optional Unit of Measure for the Observation Result
+   */
+  result: PropTypes.oneOfType([
+    valueQuantityPropShape,
+    valueStringPropShape,
+    valueNullPropShape,
+  ]),
+  /**
+   * Enum for possible Result Interpretation values (also called `Clinical Severity` and `Normalcy`).
+   * One of `'CRITICAL'`, `'EXTREMEHIGH'`, `'EXTREMELOW'`, `'PANICHIGH'`, `'PANICLOW'`, `'VABNORMAL'`, `'POSITIVE'`, `'ABNORMAL'`, `'HIGH'`, `'LOW'`, `'NORMAL'`, `'NEUTRAL'`.
+   */
+  interpretation: interpretationPropType,
+  /**
+   *  Enum for possible Result Types.
+   * One of `'ALPHA'`, `'MULTIALPHA'`, `'FREETEXT'`, `'NUMERIC'`, `'BLOODPRESSURE'`, `'CALCULATION'`, `'DATE'`, `'DATETIME'`, `'TIME'`, `'PROVIDER'`.
+   */
+  type: resultTypePropType,
+  /**
+   * Enum for possible Result Statuses.
+   * One of `'registered'`, `'preliminary'`, `'final'`, `'amended'`, `'corrected'`, `'cancelled'`, `'entered-in-error'`, `'unknown'`.
```

Comment:
Any reason why these values aren't capitalized like the other enumerated props?

Reply:
These are statuses integrated directly from HL7, vs the other enums that are all caps are from current Millennium conventions.